### ISSUE TITLE
Consider depending on Swift Metrics 2.x (whilst still allowing 1.x)

### ISSUE THREAD

**Issue by weissi** [amzn/smoke-framework#42]
[Swift Metrics 2.0](https://github.com/apple/swift-metrics/releases/tag/2.0.0) just got released. It is almost completely compatible with Swift Metrics 1.x unless you exhaustively switch on the `TimeUnit` enum. I would highly recommend depending on `swift-metrics` like the following:

```swift
// swift-metrics 1.x and 2.x are almost API compatible, so most clients should use
.package(url: "https://github.com/apple/swift-metrics.git", "1.0.0" ..< "3.0.0"),
```

alternatively, `from: "2.0.0"` will also work but may in the interim cause compatibility issues with packages that specify `from: "1.0.0"`.

If at all possible, this should be done before tagging the next release here. If using `"1.0.0" ..< "3.0.0"` this isn't even a SemVer major change because 1.x versions are still accepted.

**Comment by tachyonics** [comment]
Part of PR #45

**Comment by tachyonics** [comment]
Merged into master.
